benchmarks/benchmark.py

```diff
@@ -138,7 +139,8 @@ def run_size_scaling_benchmark(self, N_list, number=1000,
     #       problem size (down to a minimum of 1).
     #
     def run_thread_scaling_benchmark(self, N_list, number=1000,
-                                     print_stats=True):
+                                     print_stats=True, repeat=1,
+                                     on_circleci=False):
```

Comment:
See below - I'd like to use something like `limit_threads=P` or an environment variable instead of `on_circleci=False`.